earthfile2llb/interpreter.go

```diff
@@ -1170,14 +1170,16 @@ func (i *Interpreter) handleSaveImage(ctx context.Context, cmd spec.Command) err
 		return nil
 	}
 
-	labels := map[string]string{
-		"dev.earthly.version":  version.Version,
-		"dev.earthly.git-sha":  version.GitSha,
-		"dev.earthly.built-by": version.BuiltBy,
-	}
-	err = i.converter.Label(ctx, labels)
-	if err != nil {
-		return i.wrapError(err, cmd.SourceLocation, "failed to create dev.earthly.* labels during SAVE IMAGE")
+	if !opts.DisableEarthlyLabels {
```

Comment:
there should be a check to make sure the `AllowDisableEarthlyLabels` feature was enabled.

e.g.

```
if opts.DisableEarthlyLabels {
	if !i.converter.ftrs.AllowDisableEarthlyLabels {
	return i.errorf(cmd.SourceLocation, "the SAVE IMAGE --disable-earthly-labels flag must be enabled with the VERSION --allow-disable-earthly-labels feature flag.")
	}
	// deliberately don't add any labels (i.e. do nothing here)
} else {
...
}
```